it/v3/endpoints/RetrieveUkPropertyBsasControllerISpec.scala

```diff
@@ -70,13 +84,45 @@ class RetrieveUkPropertyBsasControllerISpec extends IntegrationBaseSpec {
         response.json shouldBe mtdRetrieveBsasReponseFhlJsonWithHateoas(nino, calculationId)
       }
 
-      "valid request is made and Non-FHL is returned" in new Test {
+      "valid request is made and Non-FHL is returned" in new NonTysTest {
 
         override def setupStubs(): StubMapping = {
           AuditStub.audit()
           AuthStub.authorised()
           MtdIdLookupStub.ninoFound(nino)
-          DownstreamStub.onSuccess(DownstreamStub.GET, desUrl, OK, downstreamRetrieveBsasNonFhlResponseJson)
+          DownstreamStub.onSuccess(DownstreamStub.GET, downstreamUri, OK, downstreamRetrieveBsasNonFhlResponseJson)
```

Comment:
its not your code. But the lines 
`
          AuditStub.audit()
          AuthStub.authorised()
          MtdIdLookupStub.ninoFound(nino)
`

can be moved into the request method of Test trait to avoid duplicated code